### DISCUSSION TITLE
Welcome to MAMA-multiboxing Discussions!

### DISCUSSION THREAD

**Discussion by mooreatv** [mooreatv/MAMA-multiboxing#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by redxking** [comment]
We really need to add the VFX capability. That would be a huge help for running 5 mans. I can help pay for a developer to assist if needed.
